rosplan_planning_system/src/PlanParsing/PDDLEsterelPlanParser.cpp

```diff
@@ -8,6 +8,10 @@ namespace KCL_rosplan {
 	{
 		node_handle = &nh;
 
+        // get epsilon time
+        epsilon_time = 0.1;
```

Comment:
Epsilon time is not defined in the header file - compilation fails.
Are you missing PDDLEsterelPlanParser.h from this PR?